lib/core/jdl_field.js

```diff
@@ -52,6 +52,7 @@ class JDLField {
      */
     Object.values(merged.validations).forEach(checkValidation);
     this.validations = merged.validations;
+    this.options = merged.options;
```

Comment:
Options in the JDL entity class is an array, here an object.
Second issue, I like to use objects when the target isn't a collection (if it is, I use a Set or a Map)

Reply:
Indeed, for classes it has always been array and i can't change this, without breaking everything that depends on it (this is a public API so many blueprints modules... would stop working if I change this), I also think it an array because in the beginning you only had Unary options, since we use options with keys and values an object makes more sense